kairon/shared/data/processor.py

```
@@ -9418,3 +9489,13 @@ def sanitize_query_filter(log_type: str, request) -> dict:
                 sanitized[k] = v
         return sanitized
+
+    @staticmethod
+    def get_store_page_metadata(bot: Text):
+        try:
+            metadata = StorePageMetadata.objects(bot=bot).get()
+            data = metadata.to_mongo().to_dict()
+            data.pop("_id", None)
+            return data
+        except DoesNotExist:
+            raise AppException("Store page metadata not found for this bot.")
```

Ruff (0.16.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)